fog/api/proto/view.proto

```diff
@@ -37,14 +37,8 @@ message FogViewRouterResponse {
 }
 
 message FogViewStoreDecryptionError {
-    /// The FogViewStoreUri for the specific Fog View Store that
-    /// tried to decrypt the MultiViewStoreQueryRequest and failed.
-    /// The client should subsequently authenticate with the machine
-    /// described by this URI.
-    string fog_view_store_uri = 1;
-
     /// An error message that describes the decryption error.
-    string error_message = 2;
+    string error_message = 1;
```

Comment:
This error code gets added in a follow-up [PR](https://github.com/mobilecoinfoundation/mobilecoin/pull/2353/files#diff-2195a90bc99b5988a1d8c69ea7b12e48fa76cca2dff62eb535499a618ab6af8eR45) when the MVQ Response gets refactored. So let's just hold off on adding it here since it will be added later.